easybuild/easyconfigs/a/ACTC/ACTC-1.1-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'ACTC'
+version = '1.1'
+
+homepage = 'https://sourceforge.net/projects/actc'
+description = "ACTC converts independent triangles into triangle strips or fans."
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+builddependencies = [
```

Comment:
@jdonners At the risk of sounding annoying, `(build)dependencies` is usually included below the `sources` block...

Reply:
so you just wanted the `buildopts` to be moved down below builddependencies? ok..

Reply:
Exactly.